.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+# This is a the GitHub Action to run the CI for SODA Dashboard
+
+name: SODA Dashboard CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ master ]
```

Comment:
can we skip the branches clause, as we want our CI to run on all the branches whereever there is a change, this scenario will come into play when we have some dev or temp branch for development, at that point of time CI won't run on thoose branches.